src/EnergyPlus/Coils/CoilCoolingDXCurveFitSpeed.hh

```diff
@@ -164,7 +164,7 @@ struct CoilCoolingDXCurveFitSpeed
                          const DataLoopNode::NodeData &inletNode,
                          DataLoopNode::NodeData &outletNode,
                          Real64 &PLR,
-                         int const fanOpMode,
+                         HVAC::FanOp const fanOp,
```

Comment:
Up to here things look OK (and only thru the C's). There are some comments (follow up, thoughts, etc.) but not related to these changes. I would rather see smaller scope of changes cause this is hard to review.

Reply:
I don't think you really need to review this line by line, do you? Most of the changes are either actual search-and-replace or de facto search and replace like this one. 

Basically, I change all of the `constexpr int`s to their corresponding enumerations using global search-and-replace (using `sed` if you are curious how). Then I compile. All of the `int` variables and parameters that those get assigned to show up as errors, I change all of those to the enumerated type and recompile. Then all of the variables and parameters that those things get assigned to show up as errors, and I change those types and recompile. After four of five passes of this, everything settles down.

I guess I could have done this one enumeration at a time and submitted eight PRs, but that felt like a waste of time.

TL;DR sorry. If you want to stop now, it's probably fine. Everything is green.